iml-system-docker-tests/tests/docker_integration_test.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use std::collections::{HashMap, hash_map::RandomState};
+use iml_system_test_utils::{docker, iml, vagrant, get_local_server_names};
+use iml_systemd;
+use iml_wire_types;
```

Comment:
```suggestion
```

You don't need to explicitly import these here, things will work the same without them.

Reply:
removed